packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -630,6 +679,12 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
           const { facets, state } = this;
 
           trace(`capturing oracle price `, state.updatingOracleQuote);
+          if (AmountMath.isEmpty(state.updatingOracleQuote.numerator)) {
```

Comment:
checking for empty seemed a little magic to me and I had to keep track of the Ratio getting out as an actual quote.

I thought a null value would be more clear. To test it out I made the changes locally. They passed and since I know this PR is somewhat urgent I pushed the change. I'm approving with the change. If I missed something and it's a bad idea, please re-request review and help me understand.

Reply:
thanks.